pkg/rollout/rollout.go

```diff
@@ -212,6 +221,7 @@ func (r *Rollout) nextCandidateTraffic(current int64) int64 {
 
 // updateAnnotations updates the annotations to keep some state about the rollout.
 func (r *Rollout) updateAnnotations(svc *run.Service, stable, candidate string) *run.Service {
+	r.Log.Debug("updating annotations for the service")
```

Comment:
this sounds like you're making an api call

Reply:
I also have that feeling. Is the word "updating" or the verb tense for you? For me is the word "updating"

What about "including annotations in the service"?

Reply:
I wouldn’t add logs to this method. It does the same thing always. Doesn't have different code paths. We always call this.